plugins/main/scripts/tag.py

```diff
@@ -17,7 +17,7 @@
 # Wazuh version: major.minor.patch
 version = '4.6.0'
 # App's revision number (previous rev + 1)
-revision = '01'
+revision = '02'
```

Comment:
**nipitck**: undone the change.

Reply:
Solved
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/f7278d4f2ea3bda8515259d06167fcb8c0af266e
https://github.com/wazuh/wazuh-kibana-app/pull/5696/commits/67fc2f43bdd65a0b605c9ba2b02134e00a8dfd9f